### PR TITLE
add responsive layout for 2 sections

### PR DESCRIPTION
**Pull request by dxarch**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,107 +59,120 @@
...
+          <li><a href="#" class="footer__link">Holiday homes</a></li>
+          <li><a href="#" class="footer__link">Hostels</a></li>
+        </ul>
+        <ul class="footer__list col-lg-4">
+          <li class="footer__title">Support</li>
+          <li><a href="#" class="footer__link">Contact Customer Service</a></li>
+          <li><a href="#" class="footer__link">FAQ</a></li>
+        </ul>
+      </div>
+      <p class="footer__copyright col-sm-4">© 2022 Triphouse, Inc. All rights reserved</p>
```
Comment (protokol110) [inline index.html:174]: Good homework! But in adaptive design this text stay at one  place

File: styles/styles.css [hunk 2]
```diff
@@ -36,79 +97,79 @@ body {
...
 
+.wrapper--top {
+    padding-top: 50px;
+}
+
+
 .nav {
     height: 46px;
-    padding: 51px 104px 0;
     margin-bottom: 196px;
```
Comment (protokol110) [inline styles/styles.css:115]: It's would be better to replace this margin with an accrual that is a multiple of 5

**Review by protokol110 — COMMENTED** [review]
Good homework! But need some fix

File: index.html (hunk 1 above)
Comment (dxarch) [reply index.html:174]: fixed that with margin: 0 auto

File: styles/styles.css (hunk 2 above)
Comment (dxarch) [reply styles/styles.css]: also fixed

**Pull request merged**